### ISSUE TITLE
SigmoidFocalCrossEntropy loss value is negative

### ISSUE THREAD

**Issue by KyloRen1** [tensorflow/addons#2767]
I am working on a multi-class classification model. There are 4 classes and my model outputs a probability distribution over these 4 classes using softmax. 

Training with `focal_loss = tfa.losses.SigmoidFocalCrossEntropy(from_logits=False)` I get negative values of the loss. Is it expected behavior? As I've noticed, with time the loss started to increase, from -220 at the beginning to -211

**Comment by bhack** [comment]
I suggest you to work with Keras-cv as now it was re-implemented there:
https://github.com/keras-team/keras-cv/blob/master/keras_cv/losses/focal.py

If you still have issues please open a ticket in keras-cv
